<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>938</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>938</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Sprite Editor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/src/src/icon.png</normaloff>:/src/src/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>600</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>600</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="backgroundBrush">
         <brush brushstyle="DiagCrossPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="renderHints">
         <set>QPainter::SmoothPixmapTransform</set>
        </property>
        <property name="cacheMode">
         <set>QGraphicsView::CacheNone</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTableWidget" name="frameTableWidget">
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>600</width>
          <height>180</height>
         </size>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="iconSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="rowCount">
         <number>1</number>
        </property>
        <property name="columnCount">
         <number>1</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <row/>
        <column/>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="penToolButton">
            <property name="text">
             <string>Pen</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="eraserToolButton">
              <property name="text">
               <string>Eraser</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="eraserSizeSpinBox">
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>4</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="brushToolButton">
              <property name="text">
               <string>Brush</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="brushSizeSpinBox">
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>3</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="lineToolButton">
            <property name="text">
             <string>Line</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="fillToolButton">
            <property name="text">
             <string>Fill</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="colorSelectedLayout">
            <property name="topMargin">
             <number>50</number>
            </property>
            <item>
             <widget class="ColorLabel " name="leftColorSelectedLabel">
              <property name="text">
               <string>Left Color</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ColorLabel " name="rightColorSelectedLabel">
              <property name="text">
               <string>Right Color</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="toggleGridlinesButton">
          <property name="text">
           <string>Toggle GridLines</string>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <widget class="QSlider" name="horizontalSlider">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>240</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>240</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>24</number>
          </property>
          <property name="pageStep">
           <number>6</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="fpsLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>FPS</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGraphicsView" name="animationView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>300</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>938</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="geometry">
     <rect>
      <x>344</x>
      <y>96</y>
      <width>137</width>
      <height>154</height>
     </rect>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="actionExport"/>
    <addaction name="actionSave_Current_QImage"/>
    <addaction name="actionLoad_Image"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load Project</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionSave_Current_QImage">
   <property name="text">
    <string>Save Image</string>
   </property>
  </action>
  <action name="actionLoad_Image">
   <property name="text">
    <string>Load Image</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ColorLabel </class>
   <extends>QLabel</extends>
   <header>colorlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
